README.md

```diff
@@ -43,24 +48,24 @@ Also check out the [ComposeCountryCodePicker documentation](https://jump-sdk.git
+| shape                   | Shape of the text field.                                                                                                                                                                                                                                   |
+| showCountryCode         | Whether to show the country code in the text field.                                                                                                                                                                                                        |
+| showCountryFlag         | Whether to show the country flag in the text field.                                                                                                                                                                                                        |
+| colors                  | Colors to be used for the text field. Defaults to [outlinedTextFieldColors](https://developer.android.com/reference/kotlin/androidx/compose/material/TextFieldDefaults)                                                                                    |
+| fallbackCountry         | The country to be used as a fallback if the user's country cannot be determined. Defaults to the United States. See [CountryData](https://jump-sdk.github.io/jetpack_compose_country_code_picker_emoji/ccp/com.togitech.ccp.data/-country-data/index.html) |
+| showPlaceholder         | Whether to show the placeholder number in the text field.                                                                                                                                                                                                  |
+| includeOnly             | A set of 2 digit country codes to be included in the list of countries. Set to null to include all supported countries.                                                                                                                                    |
+| clearIcon               | The icon to be used for the clear button. Set to null to disable the clear button.                                                                                                                                                                         |
+| initialPhoneNumber      | An optional phone number to be initial value of the input field.                                                                                                                                                                                           |
+| initialCountryIsoCode   | An optional ISO-3166-1 alpha-2 country code equivalent of the MCC (Mobile Country Code) of the initially selected country. This takes precedence over initialCountryPhoneCode.                                                                             |
```

Comment:
I think `initialCountryPhoneCode` is more reliable to choose initial country than `initialCountryIsoCode`

Reply:
@tanany365 my thinking here is that the Iso code is guaranteed to be unique, whereas there are some phone codes that are shared between countries. How would this impact your usecase?

Reply:
@tanany365 ok, swapped these back.
I also removed mention of MCC from the help text since that's a distinct thing from country phone codes.